operator/src/main/java/oracle/kubernetes/operator/helpers/CallBuilder.java

```diff
@@ -1907,4 +1910,31 @@ private Call readPodLogAsync(
   private CancellableCall wrap(Call call) {
     return new CallWrapper(call);
   }
+
+  public ClientPool getClientPool() {
+    return this.helper;
+  }
+
+  /**
+   * Create AccessTokenAuthentication component for authenticating user represented by
+   * the given token.
+   * @param accessToken - User's Bearer token
+   * @return - this CallBuilder instance
+   */
+  public CallBuilder withAuthentication(String accessToken) {
+    if (accessToken != null && !accessToken.isEmpty()) {
```

Comment:
We've been using the Guava method

```
Strings.isNullOrEmpty()
```

for cases like this.

Reply:
Changed to use Guava method.